packages/display/src/DisplayObject.js

```diff
@@ -84,15 +84,17 @@ export class DisplayObject extends EventEmitter
         this.renderable = true;
 
         /**
-         * The display object container that contains this display object.
+         * The display object container that contains this DisplayObject.
```

Comment:
You can make it the state of art with `{@link PIXI.DisplayObject}`

Reply:
Just in case https://github.com/microsoft/tsdoc/blob/master/spec/code-snippets/DeclarationReferences.ts